packages/components/src/components/typography/Warning/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
I'm not sure if placing this component in `typography` section makes sense.

Reply:
I think we could put it as a standalone component. I'd also didn't expect it to be placed in typography.

Reply:
@tomexx where would you put it? .)

Reply:
Let's put it as a standalone component `packages/components/src/components/Warning` we can refactor later when new elements/variations will be introduced.

Reply:
Alright. Moved in https://github.com/trezor/trezor-suite/pull/3590/commits/283b5e07c27399bcbf2dc75a637a72de62a36922. Thanks!

Resolving.